scripts/qesap/test/unit/test_qesap_configure.py

```diff
@@ -278,7 +278,7 @@ def test_configure_tfvars_overwrite_variables(configure_helper):
 
 def test_configure_create_ansible_vars(configure_helper, config_yaml_sample):
     """
-    Test that 'configure' write an azure_hana_media.yaml file in
+    Test that 'configure' write an hana_media.yaml file in
```

Comment:
s/an/a

Reply:
Thanks, I'll fix them in a different commit